.github/workflows/build.yml

```diff
@@ -9,10 +9,12 @@ on:
         description: 'Garden Linux version'
         type: string
         default: today
-      use_kms:
-        description: 'Use Garden Linux KMS'
-        type: boolean
-        default: false
+      signing_env:
```

Comment:
I personally would love to merge `signing_env`and `channel` to a more generic `target` here (as "build target") should match the intended functionality pretty well. From a development perspective `environment` is also very common for both build and release publishing.

In both cases I think it's error prone to add two new variables for the intention to:
- switch between different certificate files / sources
- change possible tags and repositories later in release publishing processes